docs/articles/documentation/extending-testcafe/creating-custom-reporter/reporterpluginhost.md

```diff
@@ -0,0 +1,280 @@
+---
```

Comment:
`reporterpluginhost.md` - we use hyphen to separate words in file names: `reporter-plugin-host.md`